src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,29 @@
+import cart from '../../assets/images/cart.svg'
```

Comment:
Configure tsconfig for better ways for import 

Should be `import cart from 'assets/images/cart.svg'`